.dockerignore

```diff
@@ -0,0 +1,59 @@
+**/node_modules
```

Comment:
@ohadlevy question: How does `.dockerignore` differ from `.gitignore`? Wouldn't we be ok with just making the `.dockerignore` a symling to `.gitignore`?

Reply:
they are nearly identical, but with minor differences afair the main differences are the bundle/*.local and .git/.gitignore etc.

in theory, we could remove the tests folder and other things for production image.
it would be nice to keep the common parts in some shared file instead.

Reply:
also, some differences in syntax that I've found out the hard way, see https://zzz.buzz/2018/05/23/differences-of-rules-between-gitignore-and-dockerignore/